Source/ACE/Entity/Landblock.cs

```diff
@@ -442,6 +442,99 @@ private void HandleGameAction(QueuedGameAction action, Player player)
                         HandleSoundEvent(obj, soundEffect);
                         break;
                     }
+                case GameActionType.PutItemInContainer:
+                    {
+                        var playerId = new ObjectGuid(action.ObjectId);
+                        var inventoryId = new ObjectGuid(action.SecondaryObjectId);
+                        if (playerId.IsPlayer())
+                        {
+                            Player aPlayer = null;
+                            WorldObject inventoryItem = null;
+
+                            if (worldObjects.ContainsKey(playerId))
+                            {
+                                aPlayer = (Player)worldObjects[playerId];
+                                inventoryItem = worldObjects[inventoryId];                                

```

Comment:
Thanks for the review.   We were chatting about it in Discord -  was going to address that later as it is such an edge case.   I would have to try to pickup something someone had dropped across a landblock boundary.   At the moment all it should do is fail to pick it up.   I can refactor to use the FindWorldObject later if that is ok?   Thanks